android/build.gradle

```diff
@@ -15,7 +15,9 @@ allprojects {
     repositories {
         google()
         mavenCentral()
-        maven { url "https://raw.githubusercontent.com/guardianproject/gpmaven/master" }
+        maven { url "https://raw.githubusercontent.com/guardianproject/gpmaven/master" 
+                url 'https://storage.googleapis.com/download.flutter.io'
```

Comment:
Why is it necessary? And why is it not set at its own `maaven` body?

Reply:
Was facing this issue when I was trying to build the app and not sure but it seems that some countries block some Google APIs as I found the solution over [here](https://stackoverflow.com/questions/59663760/flutter-gradle-not-working-except-on-https)
<img width="1383" alt="Screenshot 2023-06-11 at 6 23 19 PM" src="https://github.com/breez/breezmobile/assets/74540123/8c63c7dd-6f11-48ae-ab65-b9ba6fd47444">